src/Audit/Drupal/IniGet.php

```diff
@@ -41,6 +56,8 @@ public function audit(Sandbox $sandbox)
             return false;
         }
 
+        $this->set('local_value', $ini[$setting]['local_value']);
```

Comment:
What is this about? Perhaps some commentary would be useful here?

Reply:
That is setting the `local_value` token that the policy requires. I was getting the following error until I added that line:

```
Variable "local_value" does not exist in "success field on success.md.twig
```

Reply:
hmm, so how did policies use this in 2.x without this line present?

Reply:
`foreach ($ini[$setting] as $key => $value) { $this->set($key, $value); }`

Reply:
hmm, so how did policies use this in 2.x without this line present?

- Looks like I added it to 2.x recently 
- https://github.com/drutiny/drutiny/commit/3d91515650abba658f7d50bdcc163b6f49c52838

Reply:
foreach ($ini[$setting] as $key => $value) { $this->set($key, $value); }

- Are you saying that you want all ini variables to be added as tokens?

Reply:
So in 2.3.x, there was no strictness on policies using variables that didn't exist. But in 2.4.x there is. In the 2.3.x version of this Audit class, "local_value" was never set as a parameter which would suggest the policy got a null value when trying to use "local_value" as a variable in the success field. So then really, the policy should be updated to remove this variable rather than us support it in 2.4.x.

But if it does someone get a value for local_value, the question then is how? If this is the approach we should do that, then yes, I think all values of that setting make sense.

Reply:
doh! I just saw your comment about adding it recently to 2.x. Okay, we'll role with this then.